src/game_interpreter_battle.cpp

```diff
@@ -31,6 +31,7 @@
 #include "game_map.h"
 #include "spriteset_battle.h"
 #include <cassert>
+#include <scene_battle.h>
```

Comment:
Use ``""`` because its part of the project

Reply:
Noted!